com.powerball253.eternalmodmanager.yml

```diff
@@ -0,0 +1,56 @@
+app-id: com.powerball253.eternalmodmanager
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  # X11 stuff
+  - --share=ipc
+  - --socket=x11
+  # Needs host fs access due to Steam permissive library paths
+  # Should no longer be needed when Electron 14 becomes stable, introducing the GtkNativeFileChooser portal
+  - --filesystem=host
```

Comment:
Wouldn't `~/.var/app/com.valvesoftware.Steam` and `~/.local/share/Steam` be good enough then?

Reply:
There's also other common paths for libraries like `/media`, `/run/media` and `/mnt`, is there a way I can gain access to this paths too? Maybe use something like
```
- --filesystem=home
- --filesystem=/media
- --filesystem=/mnt
- --filesystem=/run/media
```